draft-ietf-quic-transport.md

```diff
@@ -6170,10 +6170,22 @@ considered separately.
 ### Handshake {#handshake-properties}
 
 The QUIC handshake incorporates the TLS 1.3 handshake and enjoys the
```

Comment:
enjoys is an odd word to me, but I'm struggling to find a better one.

Reply:
"benefits from" ?

Reply:
"benefits from" ?

Reply:
Ahh @ekr will be sad
But yeah “benefits from” or “inherits” are all less tongue-in-cheek

Reply:
I don't really feel strongly about it. This is the text that 8446 uses (See S 8.1), so it's not tongue in cheek. I prefer "inherits" if that's accurate.

Reply:
I also don't really care, we can keep it as is.